vllm/entrypoints/openai/tool_parsers/abstract_tool_parser.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict, List, Sequence, Union
+
+from transformers import (AutoTokenizer, PreTrainedTokenizer,
+                          PreTrainedTokenizerFast)
+
+from vllm.entrypoints.openai.protocol import (DeltaMessage,
+                                              ExtractedToolCallInformation)
+from vllm.logger import init_logger
+
+logger = init_logger(__name__)
+
+
+class ToolParser:
+    """
+    Abstract ToolParser class that should not be used directly. Provided
+    properties and methods should be used in
+    derived classes.
+    """
+
+    def __init__(self,
+                 tokenizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast,
```

Comment:
I have added a general `AnyTokenizer` type in a recent PR, perhaps you can replace this with that.

Reply:
e548b2d5